src/main/java/elise/TaskList.java

```diff
@@ -0,0 +1,157 @@
+package elise;
+
+import java.util.List;
+
+/**
+ * TaskList manages operations regarding Task.
+ */
+public class TaskList {
+
+    private final List<Task> tasks;
+
+    /**
+     * Constructor for TaskList.
+     *
+     * @param tasks Initial tasks.
+     */
+    protected TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Lists out all tasks and their status.
+     */
+    protected void list() {
+        if (tasks.size() == 0) {
```

Comment:
Nice, good to let users know the list is empty instead of displaying nothing when looping through an empty list 😩